### DISCUSSION TITLE
How to get charts via api and display it on my website？

### DISCUSSION THREAD

**Discussion by chenshaojie** [apache/superset#20182] [Q&A / Help]
I'm sorry to bother you.My English is a little poor. I'm really sorry
I want to get the chart through the API and display it on my website, but my foundation is too poor.

I tried to use the **postman** tool to get the ID and name of all charts through {/ api / v1 / chart} and tried to find the **iframe** connection, but the former succeeded and the latter couldn't find it.

So I got **access_ token** through {/ api / v1 / security / login}。 Store it as material, and I use {/ chart / {PK} / cache _ Screenshot /} but **got 404 interface**.

I haven't found a way to solve it. Do you have a way to solve this problem now?

Or you can tell me how to get the chart and display it on my website, or get the iframe automatically, instead of manually connecting through < > copy iframe on superset's website.

Thank you very much for taking the time to look at what may be a little stupid.

**Comment by yb12306** [comment]
哥们，你这个问题解决了吗？我也遇到同样问题，postman死活获取不到数据

**Comment by chenshaojie** [comment]
抱歉，我还没找到方法，我这里之前通过获取id与图表名称的时候，得到了相应图表的url，通过网址访问，可以直接到达图表的修改界面 但是我的学长拒绝了这个方法，认为这完全依赖superset网站，而不是我们自己将它集成到我们的网页上，目前还在头疼中



---Original---
From: ***@***.***&gt;
Date: Thu, Jun 9, 2022 16:56 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [apache/superset] How to get charts via api and display it on my website？ (Discussion #20182)




 
哥们，你这个问题解决了吗？我也遇到同样问题，postman死活获取不到数据
